docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,669 @@
+2. You can target any earlier version of the .NET Platform Standard to target earlier .NET platforms and versions.  Earlier .NET Platform Standard versions expose fewer APIs. Example targets:
+
+    - .NET Framework 4.6, 4.5.2, 4.5.1, or 4.5
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    [Refer to the platform mapping table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) to choose the `dotnetXX` moniker you need to target.
+    
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet54`.
+
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet54":{}
+        }
+    }
+    ```
+    
+    If you wanted to be compatible with Windows Phone Silverlight 8, you would pick `dotnet51`.  Note that this gives you the ability to target more platforms that just Windows Phone Silverlight 8, but at the cost of less available APIs.
+    
+    ```javascript
+    {
+        "frameworks":{
+            "dotnet51":{}
+        }
+    }
+    ```
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to use .NET Framework APIs not available in .NET Core, you'll have to read the next sections.
+
+## How to target .NET Framework
+
+**NOTE:** This requires .NET Framework installed on your machine.  Refer to the [Prerequisites](#prerequisites) to get dependencies installed.
+
+Keep in mind that some of the .NET Framework versions used here are no longer in support.  [Refer to this FAQ](https://support.microsoft.com/en-us/gp/framework_faq/en-us) about unsupported versions.
+
+Say you wish to support older frameworks, such as .NET Framework 4.0, so you can reach a maximum number of developers and projects.  From a project targeting perspective, this is quite simple!  You will need to specifically target the version of .NET Framework with its relevant Target Framework Monikers.  These are mapped out here:
+
+```
+.NET Framework 2.0   --> net20
+.NET Framework 3.0   --> net30
+.NET Framework 3.5   --> net35
+.NET Framework 4.0   --> net40
+.NET Framework 4.5   --> net45
+.NET Framework 4.5.1 --> net451
+.NET Framework 4.5.2 --> net452
+```
+
+For example, here's how you would write a library which targets .NET Framework 4.0:
+
+```javascript
+{
+    "frameworks":{
+        "net40":{}
+    }
+}
+```
+
+And that's it!  Although this compiled only for .NET Framework 4.0, you can use the library on newer versions of .NET Framework.
+
+If you wish to cross-compile across .NET Framework versions to use newer APIs unavailable in .NET Framework 4.0, you'll need to use `#if` directives and change your `project.json` to explicitly specify all targets and the framework assemblies which correspond to the APIs you'll use.
```

Comment:
Yeah, I think I'm going to axe that content.  After looking at it now, it doesn't appear to buy much value.